lib/rubocop/cop/style/map_into_array.rb

```diff
@@ -58,7 +58,7 @@ class MapIntoArray < Base
           [
             ^({begin kwbegin} ...)
             ({block numblock} (send !{nil? self} :each) _
-              (send (lvar _) {:<< :push :append} {send lvar begin}))
+              (send (lvar _) {:<< :push :append} !splat))
```

Comment:
You need to consider `kwsplat` as well here.

I guess anonymous forwarding too? I doubt anyone would write such code though.

Reply:
`kwsplat` gets wrapped in a `hash` (since rubocop-ast runs in legacy mode), so we're covered there

```
ruby-parse -e "arr.push(**opts)" --legacy
(send
  (send nil :arr) :push
  (hash
    (kwsplat
      (send nil :opts))))

```

anonymous forwarding is a good callout, will add